tests/torch/quantization/test_strip.py

```diff
@@ -409,3 +414,129 @@ def test_nncf_strip_lora_model(mode, decompressor_class, torch_dtype, atol, mock
             expected_class=decompressor_class,
         )
         assert torch.allclose(compressed_output, stripped_output, atol=atol)
+
+
+SIGNED_WEIGHT_SAMPLE = [-1.0, -0.75, -0.5, -0.25, 0.0, 0.25, 0.5, 0.75]
+SCALE_SAMPLE = [2.0]
+
+
+@pytest.mark.parametrize(
+    ("num_bits", "ref_scale", "torch_dtype"),
+    (
+        (4, [0.25], torch.float32),
```

Comment:
Would you kindly provide a mathematical formula with substituted values for both tests, indicating how these values should be determined? In case of changed test parameters or a new test case, give a hint to the developer to calculate these references.

Reply:
Done